app/views/prison/dashboards/search.html.erb

```diff
@@ -16,3 +16,8 @@
 
 <h3 class="heading-medium"><%= t('.processed') %></h3>
 <%= render 'processed' %>
+<script>
+  var pvb = {
```

Comment:
@chrishanes how do you feel about having this as a data attributes on the body element (or the another relevant element) rather than poluting the gloabal JS namespace?